morpheus/stages/postprocess/add_scores_stage_base.py

```diff
@@ -23,13 +23,14 @@
 from morpheus.common import TypeId
 from morpheus.config import Config
 from morpheus.messages import MultiResponseMessage
+from morpheus.pipeline.pass_thru_type_mixin import PassThruTypeMixin
 from morpheus.pipeline.single_port_stage import SinglePortStage
 from morpheus.pipeline.stream_pair import StreamPair
 
 logger = logging.getLogger(__name__)
 
 
-class AddScoresStageBase(SinglePortStage):
+class AddScoresStageBase(PassThruTypeMixin, SinglePortStage):
```

Comment:
Fair, I'm open to renaming `PassThruTypeMixin` and/or removing it. 
In an earlier iteration `PassThruTypeMixin` was doing some heavy-lifting that I didn't want to repeat, but since then its been slimmed back to a one-liner. 

The point of the mixin was just to cover the stages which receive/emit the same message type.

Reply:
Yeah I'm not sure there's anything to be done here. `PassThruTypeMixin` is a good name, afaict, and `AddScoresStageBase` is accurate, too. :shrug: